### PR TITLE
Document use_date_for_updated

### LINKED ISSUE #1097: Add docs about use_date_for_updated
https://github.com/hexojs/hexo/pull/3235

### PR DESCRIPTION
**Pull request by tomap**
+ some missing doc in other languages (highlight.*)
To fix #1097

## Check List

- [X] Others (Update, fix, translation, etc...)

### CODE REVIEW

File: source/docs/configuration.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ Setting | Description | Default
 `highlight.auto_detect` | Enable auto-detection if no language is specified | `false`
 `highlight.line_number` | Display line number | `true`
 `highlight.tab_replace` | Replace tabs by n space(s); if the value is empty, tabs won't be replaced | `''`
+`use_date_for_updated` | Use the date of the post as date of update if no updated date is provided. Useful when the file's dates are meaningless (especially when using git) | `true`
```
Comment (curbengh) [inline source/docs/configuration.md]: Shouldn't this be under `Date / Time format` section? It's under similar section in [default_config.js](https://github.com/tomap/hexo/blob/677f5b86069a3395e630d749bd3dd7c3a30d3ba8/lib/hexo/default_config.js#L49).

**Comment by SukkaW** [comment]
@curbengh move to Date section for other languages as well?

File: source/ko/docs/configuration.md [hunk 2]
```diff
@@ -54,6 +54,11 @@ title: Configuration
 `relative_link` | 루트 폴더에 대한 상대 경로로 링크를 만들 것인지 선택 | `false`
 `future` | 미래의 포스트를 표시할 것인지 선택 | `true`
 `highlight` | Code block의 설정 |
+`highlight.enable` | Enable syntax highlight | `true`
+`highlight.auto_detect` | Enable auto-detection if no language is specified | `false`
+`highlight.line_number` | Display line number | `true`
+`highlight.tab_replace` | Replace tabs by n space(s); if the value is empty, tabs won't be replaced | `''`
+`use_date_for_updated` | Use the date of the post as date of update if no updated date is provided. Useful when the file's dates are meaningless (especially when using git) | `true`
```
Comment (SukkaW) [inline source/ko/docs/configuration.md]:
```suggestion
```
Remove duplicated.

File: source/pt-br/docs/configuration.md [hunk 3]
```diff
@@ -55,6 +55,11 @@ Configuração | Descrição | Padrão
 `relative_link` | Links para o diretório raiz? | `false`
 `future` | Exibir postagens futuras? | `true`
 `highlight` | Configurações de bloco de código |
+`highlight.enable` | Enable syntax highlight | `true`
+`highlight.auto_detect` | Enable auto-detection if no language is specified | `false`
+`highlight.line_number` | Display line number | `true`
+`highlight.tab_replace` | Replace tabs by n space(s); if the value is empty, tabs won't be replaced | `''`
+`use_date_for_updated` | Use the date of the post as date of update if no updated date is provided. Useful when the file's dates are meaningless (especially when using git) | `true`
```
Comment (SukkaW) [inline source/pt-br/docs/configuration.md]:
```suggestion
```
Remove duplicated.

File: source/ru/docs/configuration.md [hunk 4]
```diff
@@ -54,6 +54,11 @@ title: Конфигурация
 `relative_link` | Создание ссылок относительно корневой папки? | `false`
 `future` | Отображать будущие посты? | `true`
 `highlight` | Настройки блоков кода |
+`highlight.enable` | Enable syntax highlight | `true`
+`highlight.auto_detect` | Enable auto-detection if no language is specified | `false`
+`highlight.line_number` | Display line number | `true`
+`highlight.tab_replace` | Replace tabs by n space(s); if the value is empty, tabs won't be replaced | `''`
+`use_date_for_updated` | Use the date of the post as date of update if no updated date is provided. Useful when the file's dates are meaningless (especially when using git) | `true`
```
Comment (SukkaW) [inline source/ru/docs/configuration.md]:
```suggestion
```
Remove duplicated.

File: source/th/docs/configuration.md [hunk 5]
```diff
@@ -55,6 +55,11 @@ Setting | Description | Default
 `relative_link` | Make links relative to the root folder? | `false`
 `future` | Display future posts? | `true`
 `highlight` | Code block settings |
+`highlight.enable` | Enable syntax highlight | `true`
+`highlight.auto_detect` | Enable auto-detection if no language is specified | `false`
+`highlight.line_number` | Display line number | `true`
+`highlight.tab_replace` | Replace tabs by n space(s); if the value is empty, tabs won't be replaced | `''`
+`use_date_for_updated` | Use the date of the post as date of update if no updated date is provided. Useful when the file's dates are meaningless (especially when using git) | `true`
```
Comment (SukkaW) [inline source/th/docs/configuration.md]:
```suggestion
```
Remove duplicated.

File: source/zh-cn/docs/configuration.md [hunk 6]
```diff
@@ -76,6 +76,11 @@ skip_render: "_posts/test-post.md"
 `relative_link` | 把链接改为与根目录的相对位址 | false
 `future` | 显示未来的文章 | true
 `highlight` | 代码块的设置 |
+`highlight.enable` | Enable syntax highlight | `true`
+`highlight.auto_detect` | Enable auto-detection if no language is specified | `false`
+`highlight.line_number` | Display line number | `true`
+`highlight.tab_replace` | Replace tabs by n space(s); if the value is empty, tabs won't be replaced | `''`
+`use_date_for_updated` | Use the date of the post as date of update if no updated date is provided. Useful when the file's dates are meaningless (especially when using git) | `true`
```
Comment (SukkaW) [inline source/zh-cn/docs/configuration.md]:
```suggestion
```
Remove duplicated.

File: source/zh-tw/docs/configuration.md [hunk 7]
```diff
@@ -58,6 +58,11 @@ title: 配置
 `relative_link` | 把連結改為與根目錄的相對位址 | `false`
 `future` | 顯示未來的文章 | `true`
 `highlight` | 程式碼區塊的設定 |
+`highlight.enable` | Enable syntax highlight | `true`
+`highlight.auto_detect` | Enable auto-detection if no language is specified | `false`
+`highlight.line_number` | Display line number | `true`
+`highlight.tab_replace` | Replace tabs by n space(s); if the value is empty, tabs won't be replaced | `''`
+`use_date_for_updated` | Use the date of the post as date of update if no updated date is provided. Useful when the file's dates are meaningless (especially when using git) | `true`
```
Comment (SukkaW) [inline source/zh-tw/docs/configuration.md]:
```suggestion
```
Remove duplicated.

**Pull request merged**